library/catalog/tests/test_models.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from ..models import MongoUser, DjangoUser
+
+
+class MongoUserTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        user = MongoUser()
+        user.first_name = 'test'
+        user.last_name = 'test'
+        user.username = 'test_user'
+        user.email = 'test_user@gmail.com'
+        user.password = 'test1234'
+        user.save()
```

Comment:
Do we use a real database for testing ?

Reply:
yes. But the user is deleted after the test